vignettes/miaViz.Rmd

```diff
@@ -98,6 +98,8 @@ plots$abundance / plots$SampleType +
      plot_layout(heights = c(9, 1))
 ```
 
+Further example about composition barplot can be found at Orchestrating Microbiome Analysis [@OMA]; Microbiome Community.
```

Comment:
Remove "Microbiome Community" (that, or relevant equivalent is already in the reference.